docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -464,6 +464,82 @@ public static class MyCustomizer implements HttpServerOptionsCustomizer {
 <1> By making the class a managed bean, Quarkus will take the customizer into account when it starts the Vert.x servers
 <2> In this case, we only care about customizing the HTTP server, so we just override the `customizeHttpServer` method, but users should be aware that `HttpServerOptionsCustomizer` allows configuring the HTTPS and Domain Socket servers as well
 
+=== SSL customizations
+
+Quarkus has simple ssl configuration available as shown in section xref:ssl[]. However, it can be customized programmatically for advanced use cases such as SSL Hot Reloading, using JDK, OS and custom trusted Certificate Authorities or using PEM files. Below is an example code snippet which allows to supply a custom `X509KeyManager` and `X509TrustManager` with reloading enabled. Third party libraries such as link:https://github.com/Hakky54/sslcontext-kickstart[SSLContext-Kickstart] can provide these kind of custom ssl objects.
```

Comment:
I know it still uses `ssl`, but let's use TLS. SSL is deprecated for more than 10 years now

Reply:
As Vale suggested, use `SSL/TLS`